### ISSUE TITLE
Details of the pretrain procedure

### ISSUE THREAD

**Issue by XJay18** [mabaorui/OnSurfacePrior#7]
Hi, thanks for your code! I am working on custom datasets and wonder could you please give me some instructions on pretraining the on-surface prior (i.e., [pointnet](https://github.com/mabaorui/OnSurfacePrior/blob/master/onSurPrior.py#L546) and [local_decoder](https://github.com/mabaorui/OnSurfacePrior/blob/master/onSurPrior.py#L561)) using custom datasets. To be specific, how to sample query points and construct the groundtruth labels (i.e., udf) for training the two networks. 
Thanks in advance and waiting for your answer.
